src/openrct2-ui/windows/Guest.cpp

```diff
@@ -1317,26 +1317,26 @@ void window_guest_stats_paint(rct_window* w, rct_drawpixelinfo* dpi)
     int32_t happiness = peep->Happiness;
     if (happiness < 10)
         happiness = 10;
-    int32_t ebp = COLOUR_BRIGHT_GREEN;
+    int32_t barColor = COLOUR_BRIGHT_GREEN;
```

Comment:
Should be `barColour`, as we use British English (applies to whole diff)

Reply:
As a Canadian, that's a rather embarrassing oversight. Fixed all.